test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -419,6 +419,29 @@ public void AddPicture_should_not_duplicate_the_image_source_When_the_same_image
         imageParts.Count.Should().Be(1);
     }
     
+    [Test]
+    public void AddPicture_should_not_duplicate_the_png_image_source_When_the_same_svg_image_is_added_a_second_apart()
```

Comment:
Please move this test into the test project `ShapeCrawler.Tests.Load`. Here is [Why](https://github.com/ShapeCrawler/ShapeCrawler/pull/885#discussion_r1913705675).

Reply:
I've moved both the svg and raster tests across.